packages/core/admin/admin/src/content-manager/hooks/useFetchContentTypeLayout/utils/formatLayouts.js

```diff
@@ -164,6 +164,27 @@ const formatListLayoutWithMetas = contentTypeConfiguration => {
       return acc;
     }
 
+    if (type === 'component') {
+      const component = components[fieldSchema.component];
+      const mainFieldName = component.settings.mainField;
+      const mainFieldAttributes = component.attributes[mainFieldName];
```

Comment:
`mainFieldAttribute` it should be singular.

Reply:
I changed that in eaf8baff85135d95d565378903e8b9c802572971. I don't agree, because I still think `value`, `min`, `max` ... are attributes (plural) of the mainField, but you know the data-structure better than me and I think it's not worth to discuss that any further. 👍🏼